analysis_options.yaml

```diff
@@ -9,94 +9,7 @@ analyzer:
 
 linter:
   rules:
-    - always_put_required_named_parameters_first
-    - avoid_annotating_with_dynamic
```

Comment:
why remove these? Sometimes it's nice to check more than what we recommend!

Reply:
Because we have a single recommended style now, so I'd like to use that. I don't like fragmentation.

If you feel these should be in package:lints, please file tracking issues for them there.

Reply:
The bar is CRAZY high for getting a lint approved across the bar. And a lot of these are really nice!

Reply:
No, the bar isn't that high actually, especially for recommended lints (it's a bit higher for scoring).

About the two specific lints you mention here, looks like neither of them were in the triage sheet when we selected the initial set, so I suspect they are fairly new. So it would make a lot of sense to log bugs for them, so we can consider them for inclusion in package:lints v1.1. Can you please file here?
https://github.com/dart-lang/lints/issues?q=is%3Aissue+is%3Aopen+label%3Atype-lint